src/libbson/tests/json/bson_corpus/string.json

```diff
@@ -21,12 +21,12 @@
         {
             "description": "two-byte UTF-8 (\u00e9)",
             "canonical_bson": "190000000261000D000000C3A9C3A9C3A9C3A9C3A9C3A90000",
-            "canonical_extjson": "{\"a\" : \"éééééé\"}"
+            "canonical_extjson": "{\"a\" : \"\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\"}"
```

Comment:
These tests are lumped in with the `code.json` skips and [CDRIVER-4017](https://jira.mongodb.org/browse/CDRIVER-4017).